backend-application/build.gradle

```diff
@@ -11,4 +11,6 @@ dependencies {
 
     compile libraries.persistence.hibernateCore
     compile libraries.persistence.h2Driver
+
+    compile libraries.dataformat.jacksonDatatype
```

Comment:
please indicate that it's not a generic "datatype" but java8time. e.g. `jacksonDatatypeJsr310`